src/app/child-dev-project/notes/model/note.spec.ts

```diff
@@ -118,9 +145,21 @@ describe("Note", () => {
 
   it("should return colors", function () {
     const note = new Note("1");
-    note.category = { name: "test", color: "#FFFFFF" };
+    note.category = { id: "", label: "test", color: "#FFFFFF" };
     expect(note.getColor()).toBe("#FFFFFF");
     note.warningLevel = WarningLevel.URGENT;
     expect(note.getColor()).toBe(WarningLevelColor(WarningLevel.URGENT));
   });
+
+  it("transforms interactionType from config", function () {
```

Comment:
I had added this as an explicit test case before changing the implementation in the sense of test driven development because the general schema test was ignoring the special transformation before